common/src/main/java/com/redhat/cloud/notifications/XRhIdentityUtils.java

```diff
@@ -21,6 +22,10 @@ public static String generateEncodedXRhIdentity(final String orgId) {
         final JsonObject json = new JsonObject();
         json.put("identity", identity);
 
-        return Base64Utils.encode(json.toString());
+        // Log the org ID and the contents of the x-rh-identity header we are about to send..
+        final String generatedXRhIdentityContents = Base64Utils.encode(json.toString());
+        Log.debugf("the encoded org ID is '%s', and the encoded x-rh-identity contents are '%s'", orgId, generatedXRhIdentityContents);
```

Comment:
Quarkus doesn't seem happy with `Log` in this class. Maybe that's because the method is `static`.

You can replace `Log` with a `org.jboss.logging.Logger`:
```java
private static final Logger LOGGER = Logger.getLogger(XRhIdentityUtils.class);
```

Reply:
My bad, the failure comes from the test: it is not a `@QuarkusTest` so `Log` doesn't work. Building the `Logger` as suggested above should work.

Reply:
Arg, I'm sorry. I thought that it would be interesting to log the endpoint's ID too and I ended up adding the logs to the secrets utils... I don't know if you're okay with it or you prefer the other version

Reply:
I'm facing the same error, let me fix it